crates/red_knot_python_semantic/src/types/property_tests/type_generation.rs

```diff
@@ -167,15 +167,15 @@ impl Ty {
                     .symbol
                     .expect_type()
                     .expect_class_literal()
-                    .class,
+                    .default_specialization(db),
```

Comment:
Probably we should have a TODO for better coverage of generic types (including non-default specializations) in the property tests?

Reply:
Done